subsys/bluetooth/mesh/Kconfig

```diff
@@ -103,6 +103,10 @@ menuconfig BT_MESH_ADV_EXT
 
 if BT_MESH_ADV_EXT
 
+config BT_MESH_RELAY_ADV_SET_ENABLED
```

Comment:
I'm strictly against such Kconfig options. Lets solve this in code instead and keep Kconfig clean as much as possible. You can do this, for example:
```
#ifdef CONFIG_BT_MESH_RELAY_ADV_SETS
#define RELAY_ADV_SETS CONFIG_BT_MESH_RELAY_ADV_SETS
#else
#define RELAY_ADV_SETS 0
#endif
```